plugins/webhooks/index.js

```diff
@@ -421,14 +459,14 @@ function pullRequestEvent(pluginOptions, request, reply, parsed) {
                         restrictPR
                     };
 
+                    updateAdmins(userFactory, username, scmContext, pipeline);
```

Comment:
We do not need await here? I am worried about when `updateAdmins` will be completed.

Reply:
You are right. I add `await`. Thank you.